### ISSUE TITLE
IRC information in docs is incorrect

### ISSUE THREAD

**Issue by jkugler** [jordansissel/fpm#1998]
On page https://github.com/jordansissel/fpm/blob/v1.15.1/docs/contributing.rst its says you can "join the IRC channel #fpm on irc.freenode.org." #fpm on freenode is now the "Fortran package manager," and #fpm on LibreChat is an unclaimed channel with less than five people in it. :)

**Comment by jordansissel** [comment]
Good catch. It’s been a while since I’ve been on IRC. I’m ok to remove the reference in the fpm docs 👍

**Comment by jordansissel** [comment]
Fixed in #2109 👍
